plugins/cpp_metrics/parser/src/cppmetricsparser.cpp

```diff
@@ -0,0 +1,78 @@
+#include <cppmetricsparser/cppmetricsparser.h>
+
+#include <model/cppastnodemetrics.h>
+#include <model/cppastnodemetrics-odb.hxx>
+#include <model/cppdirectorymetrics.h>
+#include <model/cppdirectorymetrics-odb.hxx>
+
+#include <boost/filesystem.hpp>
+
+#include <util/logutil.h>
+
+#include <memory>
+
+namespace cc
+{
+namespace parser
+{
+
+CppMetricsParser::CppMetricsParser(ParserContext& ctx_): AbstractParser(ctx_)
+{
+}
+
+bool CppMetricsParser::accept(const std::string& path_)
+{
+  std::string ext = boost::filesystem::extension(path_);
+  return ext == ".dummy";
+}
+
+bool CppMetricsParser::parse()
+{        
+  for(std::string path : _ctx.options["input"].as<std::vector<std::string>>())
+  {
+    if(accept(path))
+    {
+      LOG(info) << "CppMetricsParser parse path: " << path;
+    }
+  }
+  return true;
+}
```

Comment:
I guess this is just some copied code from the *dummy* plugin.

Reply:
I added the function parameter counting metric.